app/services/subscriptions/override_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Subscriptions
+  class OverrideService < BaseService
+    def call(plan_args:, subscription_args:)
+      ActiveRecord::Base.transaction do
+        plan_result = ::Plans::CreateService.new.create(**plan_args)
+
+        return plan_result unless plan_result.success?
+
+        subscription_result = ::Subscriptions::CreateService.new.create(
+          **subscription_args.merge(plan_id: plan_result.plan.id)
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)